src/parser/parse_tree_cl.c

```diff
@@ -18384,3 +18400,102 @@ pt_print_dblink_table (PARSER_CONTEXT * parser, PT_NODE * p)
 
   return q;
 }
+
+
+
+static PT_NODE *
+pt_apply_create_server (PARSER_CONTEXT * parser, PT_NODE * p, void *arg)
+{
+  PT_APPLY_WALK (parser, p->info.create_server.server_name, arg);
+  /*
+     PT_NODE *server_name;
+     PT_NODE *host;
+     PT_NODE *port;  
+     PT_NODE *dbname;  
+     PT_NODE *user;
+     PT_NODE *pwd;
+     PT_NODE *prop;
+     PT_NODE *comment;
+   */
+  return p;
+}
+
+static PARSER_VARCHAR *
+pt_print_create_server (PARSER_CONTEXT * parser, PT_NODE * p)
+{
+  PARSER_VARCHAR *q = 0, *r;
+  PT_CREATE_SERVER_INFO *si = &(p->info.create_server);
+
+  q = pt_append_nulstring (parser, q, "CREATE SERVER ");
+  q = pt_append_nulstring (parser, q, (char *) si->server_name->info.name.original);
+  q = pt_append_nulstring (parser, q, " ( HOST=");
+  if (si->host->node_type == PT_VALUE)
+    {
+      q = pt_append_bytes (parser, q, (char *) si->host->info.value.data_value.str->bytes,
+			   si->host->info.value.data_value.str->length);
+    }
+  else
+    {
+      q = pt_append_nulstring (parser, q, (char *) si->host->info.name.original);
+    }
+
+  q = pt_append_nulstring (parser, q, ", PORT=");
+  r = pt_print_bytes (parser, si->port);
+  q = pt_append_varchar (parser, q, r);
+
+  q = pt_append_nulstring (parser, q, ", DBNAME=");
+  q = pt_append_nulstring (parser, q, (char *) si->dbname->info.name.original);
+
+  q = pt_append_nulstring (parser, q, ", USER=");
+  q = pt_append_nulstring (parser, q, (char *) si->user->info.name.original);
+
+  if (si->pwd != NULL)
+    {
+      q = pt_append_nulstring (parser, q, ", PASSWORD=");
+      r = pt_print_bytes (parser, si->pwd);
+      q = pt_append_varchar (parser, q, r);
+    }
+
+  if (si->prop != NULL)
+    {
+      q = pt_append_nulstring (parser, q, ", PROPERTIES=");
+      r = pt_print_bytes (parser, si->prop);
+      q = pt_append_varchar (parser, q, r);
+    }
+#if 0
+  if (si->comment != NULL)
```

Comment:
Why is supposed to "#if 0" ?

Reply:
I wasn't sure there was a need to print the comments.

Reply:
The comment content has been modified to be printed as well.